manifests/ruby.yml

```diff
@@ -257,6 +257,8 @@ tests/:
       Test_RemoteConfigurationUpdateSequenceFeaturesNoCache: missing_feature
       Test_RemoteConfigurationUpdateSequenceLiveDebugging: missing_feature
       Test_RemoteConfigurationUpdateSequenceLiveDebuggingNoCache: missing_feature
+  test_data_integrity.py:
+    Test_LibraryHeaders: missing_feature
```

Comment:
By doing that, you deactivate the other test in `Test_LibraryHeaders` (`test_datadog_container_id`).

You can either : 

* do another test class, and declar it that way in the manifest
* or decorate the new `test_datadog_entity_id` with `@missing_feature`.

It does not make any difference on the execution and test results, it's just up to you to decide what is the way you're more confortable with.